playwright-e2e/dsm/pages/tablist/sequencing-order-tab.ts

```diff
@@ -1,23 +1,25 @@
 import { Locator, Page, expect } from '@playwright/test';
 import { getDate } from 'utils/date-utils';
+import TabBase from './tab-base';
+import { Tab } from 'dsm/enums';
 
-export default class SequeuncingOrderTab {
+export default class SequeuncingOrderTab extends TabBase {
   private readonly SAMPLE_ROW_XPATH = '//app-sequencing-order//tr';
   private readonly DATE_FIELD_XPATH = `//input[@data-placeholder='mm/dd/yyyy']`;
 
-  constructor(private readonly page: Page) {
+  constructor(page: Page) {
+    super(page, Tab.SEQUENCING_ORDER);
   }
 
   /* Actions */
 
   public async waitForReady(): Promise<void> {
-    const sequencingTab = this.page.locator(`//tab[@heading='Sequencing Order']`);
-    await sequencingTab.scrollIntoViewIfNeeded();
-    await expect(sequencingTab, 'The Sequencing Tab is not currently active').toHaveClass(/active/);
+    await this.toLocator.scrollIntoViewIfNeeded();
+    await expect(this.toLocator, 'The Sequencing Tab is not currently active').toHaveClass(/active/);
```

Comment:
`TabBase.ts` takes care constructing the tab xpath automatically. Use `toLocator` to find elements inside the opened tab.